packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -101,7 +101,9 @@ export default (new Bundler({
           for (let asset of assets) {
             let bundle = bundleGraph.createBundle({
               entryAsset: asset,
-              isEntry: asset.isInline ? false : Boolean(dependency.isEntry),
+              isEntry: asset.isInline
```

Comment:
I think we may want to do something similar here in the future – `isEntry` on a bundle really means don't include a content hash.